Libs/RaptureCore/src/integrationtest/java/rapture/kernel/IndexApiImplTest.java

```diff
@@ -0,0 +1,288 @@
+package rapture.kernel;
```

Comment:
the path of this file looks wrong.  it should be in integrationTest, not integrationtest